### DISCUSSION TITLE
Add more examples to gallery - Matplotlib or Seaborn

### DISCUSSION THREAD

**Discussion by Casper-Guo** [theOehrly/Fast-F1#359] [General]
I have built a lot of visualizations for my own use that can be easily rewritten as examples. Most of those are done in Seaborn, however. If  the maintainer wants to keep things uniform, I can refactor most of them into basic matplotlib instead.

**Comment by theOehrly** [comment]
This is purely a question of style and consistency, I'd assume. Seaborn itself should work perfectly fine with the examples gallery, although I haven't tested it.

I'd say, in general, if it is beneficial to use Seaborn for a specific task (faster, simpler, more concise, ...) then it is worth showing that in the example. 
On the other hand, it is good to have consistency in the examples, as you point out. But if that means that the code will become much more complex to understand, it is likely not worth it. The only people that would greatly benefit from consistency with respect to using Matplotlib or Seaborn are newcomers to Python, at least in my opinion. But those people will also have great difficulty with understanding complex examples.

So in the end, it comes down to the specific example I'd say. If it is easily accomplished with basic matplotlib then that should be used. But if Seaborn makes the example more concise and easy to understand, use Seaborn. I'll let you judge what would be better.

Also, for much of the same reasons, there's already one example that uses Plotly (not yet in the published docs). Plotly is used there because it would be a hugely complex task to create this kind of visualization in Matplotlib.

I'm looking forward to your PR(s) for adding more examples 😃👍
